src/Views/Content.elm

```diff
@@ -5,7 +5,7 @@ module Views.Content exposing (view)
 
 import Assets exposing (AssetPath(..), path)
 import CallToAction exposing (CallToAction, callToActionNoDesktopButton)
-import Html exposing (Html, a, article, button, div, h2, img, p, section, span, text, ul)
```

Comment:
is the button no longer needed?

Reply:
Yeah - I guess maybe it was only being used for the 'Keep reading' button on the Timeout page.